data/styles/clear/include/Icons.mapcss

```diff
@@ -426,12 +426,10 @@ node[historic=castle],
 area[historic=castle]
 {z-index: 2890;}
 
-node[amenity=hospital],
 node[building=hospital],
 area[building=hospital],
 node[building=university],
 area[building=university],
-area[amenity=hospital],
```

Comment:
z-index for area fill is set in Basemap.mapcss;
here it was added to raise the priority of icon/text I guess, thus overriding and breaking fill priorities;
also it's impossible to set different z-indices for area and node